### DISCUSSION TITLE
Production sites?

### DISCUSSION THREAD

**Discussion by osseonews** [vikejs/vike#1383] [Discussion]
Just got Vike working in cloudflare workers in a few minutes, so I'm pretty sure this is a very promising project that I am hoping can eventually replace NextJs (my issues with Nextjs right now are manifold) and greatly simplify our react-based sites. The simplicity of Vike is a breath of fresh air when compared to things like NextJs. My question is: Are there any production sites that use Vike currently? Any live examples? What are the risks in trying to bring this to a real production environment?

**Comment by brillout** [comment]
Yes. Let me elaborate by adding an entry to https://vike.dev/faq later today / this week.

**Reply by brillout** [reply]
Done: 28c03b899ac16c259707eee262e925d6589b7f41. (Rendered: https://vike.dev/faq.)

As mentioned in the new FAQ entries: would your company be up for [sponsoring](https://github.com/vikejs/vike/issues/1350)? That's the most direct way you can help Vike thrive.

**Reply by osseonews** [reply]
Thanks! Trying to first see if we can develop an MVP with Vike. We are trying to build an e-commerce application. If we can get the MVP, then certainly we will sponsor.

**Reply by brillout** [reply]
> If we can get the MVP, then certainly we will sponsor.

Thank you, we appreciate it.

Let us know how it goes. Also let us know if you run into any blocker (even though given Vike's design it's fairly unlikely).

(And in case you want some extra help @AurelienLourot (Vike core team member) is providing consulting.)
